test/java/seedu/addressbook/utils/UtilsTest.java

```diff
@@ -0,0 +1,147 @@
+package seedu.addressbook.utils;
+
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+import org.junit.Test;
+
+import seedu.addressbook.common.Utils;
+
+public class UtilsTest {
+
+	/* 
+	 * 
+	 * Test isAnyNull function 
+	 * 
+	 * */
+	@Test
+	public void isAnyNull_oneInt() {
+		// False since no Null
+		assertFalse(Utils.isAnyNull(2));
+	}
+	
+	@Test
+	public void isAnyNull_fewInts() {
```

Comment:
the test name should have 3 parts `<method being tested>_<what is being tested>_<what is the expected return value>`
you can eliminate part 3 and 2 only if you have broadened the scope of the test to include testing multiple things.